Formula/nwchem.rb

```diff
@@ -40,16 +41,18 @@ def install
       inreplace "util/util_nwchemrc.F", "/etc/nwchemrc", "#{etc}/nwchemrc"
 
       ENV["NWCHEM_TOP"] = buildpath
-      ENV["PYTHONVERSION"] = "2.7"
-      pyhome = `python-config --prefix`.chomp
-      ENV["PYTHONHOME"] = pyhome
       ENV["NWCHEM_LONG_PATHS"] = "Y"
+      # needed to use python 3.X to skip using default python2
+      ENV["PYTHONVERSION"] = [
+        `python3 -c 'import sys;print(sys.version_info.major)'`.chomp,
+        `python3 -c 'import sys;print(sys.version_info.minor)'`.chomp,
+      ].join(".")
```

Comment:
If you want to autodetect it then 
```suggestion
      ENV["PYTHONVERSION"] = Language::Python.major_minor_version "python3"
```
probably works

Reply:
Applied with commit c7018e675938cb162b0329a198c84cdc78c179ad